clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/CatsSearchProvider.groovy

```diff
@@ -93,7 +93,7 @@ class CatsSearchProvider implements SearchProvider {
     }
 
     SearchResultSet resultSet = new SearchResultSet(
-      totalMatches: matches.size(),
+      totalMatches: results.size(),
```

Comment:
(nothing in the UI is using that field value, not sure if any external clients are using it)